src/utilities/service-utils.ts

```diff
@@ -129,7 +129,7 @@ const _mapPagedAxiosResponse = <TRecord>(
     // Ensure result data is wrapped within records
     let resultObjects;
     let rowCount = 0;
-    if (CollectionUtils.hasValues(data?.resultObject)) {
+    if (Array.isArray(data?.resultObject)) {
```

Comment:
Could we keep the `CollectionUtils` call for consistency with the rest of the code base? I believe the real issue is that we're not initializing `resultObjects` on line 130

Reply:
the problem with initializing it on line 130 is that if the `response.data` is null, `resultObjects` wouldn't be null then (which it should be in that case). Although I guess I could just manually null-check `response.data`

Reply:
Right - I think the larger point I'm trying to get at is what the value of retaining a `null` or `undefined` value for `resultObjects` is. For a list/index API call, it would ease the developer burden of null-checking along the way if we could bank on `resultObjects` always being an initialized array, even if it is empty.

If we're already in here updating it, it's something that's come up before (https://github.com/AndcultureCode/AndcultureCode.JavaScript.Core/pull/16#issuecomment-653090064)

We don't have to do it now, but I'm not sure if this PR as-is introduces any significant benefit

Reply:
@brandongregoryscott read the comment on #16, makes sense. I'll just update it to always assign it to an empty array then.

Reply:
Thanks for making that update @mrjones2014 👍 I think there are some references left that still need to be updated, since the build failed after that push